frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pages/employee/NewApplication/PropertyDetailsTableSection.js

```
@@ -0,0 +1,348 @@
+      </table>
+
+      {formErrors.totalUnitArea && (
+
+        <p style={{ color: "red", fontSize: "14px", padding: "4px 8px", textAlign: "right" }}>
+          {formErrors.totalUnitArea}
+        </p>
+
+      )}
+      <div style={{ textAlign: "right", marginTop: "0.5rem" }}>
+        <a href="#" style={styles.addMoreLink} onClick={(e) => { e.preventDefault(); addUnit(); }}>
+          {t("Add more")}
```

Biome (1.9.4)

[error] Use a button element instead of an a element.

Anchor elements should only be used for default sections or page navigation
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)